pkg/scheduler/core/generic_scheduler.go

```diff
@@ -411,7 +409,7 @@ func (g *genericScheduler) prioritizeNodes(
 				for i := range *prioritizedList {
 					host, score := (*prioritizedList)[i].Host, (*prioritizedList)[i].Score
 					if klog.V(10).Enabled() {
-						klog.Infof("%v -> %v: %v, Score: (%d)", util.GetPodFullName(pod), host, g.extenders[extIndex].Name(), score)
+						klog.InfoS("Pod score", "pod", klog.KObj(pod), "host", host, "extenders name", g.extenders[extIndex].Name(), "score", score)
```

Comment:
Active voice: `Extender scored node for pod"

Reply:
I'll fix it, thanks.